manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+PackageLocale: en-US
+Publisher: Wacom Technology, Corp.

```

```suggestion
Publisher: Wacom Technology Corp.
```
to match Control Panel.